templates/asp-app.json

```diff
@@ -189,7 +189,7 @@
         },
         {
           "name": "[parameters('additional_host_name')]",
-          "condition": "[not(equals(parameters('is_frontend'), '0'))]",
+          "condition": "[not(contains(createArray('', 'null', 'false'), parameters('additional_host_name')))]",
```

Comment:
does this work if I have `myfalsewebsite.platform.hmcts.net` ?
is the contains an equality check in the array?

Reply:
Yes, it's an equality check.

The condition will return true when additional_host_name is not one of "", "null", "false".